components/scream/src/mct_coupling/tests/CMakeLists.txt

```diff
@@ -19,15 +19,12 @@ set (SCREAM_INCLUDE
      ${SCREAM_TPL_INCLUDE_DIRS}
      ${CSM_SHARE}
 )
-message("ASD")
-message("${ARRAY_SCORPIO_SRCS}")
-message("${SCREAM_LIBS}")
-message("${SCREAM_INCLUDE}")
 # Test atmosphere processes
 EkatCreateUnitTest(scorpio_interface "${ARRAY_SCORPIO_SRCS}" 
   LIBS ${SCREAM_LIBS} csm_share
   LIBS_DIRS ${SCREAM_INCLUDE} ${CSM_SHARE}
   INCLUDE_DIRS ${SCREAM_INCLUDE}
+  THREADS 1 4
```

Comment:
I don't think we want to run I/O threaded. Perhaps you meant to use MPI_RANK?

Reply:
yes, you are right.  I'll make that change.  Is there anything special that I have to do in CMAKE to make sure that MPI is enabled for testing?  I see in the build log that openmp is enabled but I'm not sure that mpicxx is being used.

Reply:
MPI is required. Not really explicitly, but implicitly. At the beginning of our main CMakeLists.txt, we disable MPI C++ bindings; in order to do this, we try to detect what is the MPI distribution name (openmpi vs mpich, e.g.)., and if we cannot detect the MPI distro name, we error out.

**tl;dr**: you can assume MPI is _always_ available.